pkg/controllers/management/nodedriver/machine_driver.go

```diff
@@ -61,7 +61,7 @@ func (m *Lifecycle) download(obj *v3.NodeDriver) (*v3.NodeDriver, error) {
 		return obj, nil
 	}
 
-	_, err = v3.NodeDriverConditionDownloaded.Do(obj, func() (runtime.Object, error) {
+	newObj, err := v3.NodeDriverConditionDownloaded.Once(obj, func() (runtime.Object, error) {
```

Comment:
why it is changed from Do to Once?

Reply:
if may error after download and need retry, when retry, the binary already download, we don't need to do it again, so I change to once.